resolwe_bio/processes/alignment/bwamem2.yml

```diff
@@ -0,0 +1,264 @@
+# =============
+# Aligner - BWA-MEM2
+# =============
+---
+
+- slug: alignment-bwa-mem2
+  name: BWA MEM2
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: public.ecr.aws/genialis/resolwebio/dnaseq:6.1.0
+    resources:
+      cores: 20
+      memory: 16384
```

Comment:
Set the requirements to:

cores: 4
memory: 32768

In this way, the process will work for large inputs. Please modify this also for the original BWA MEM process.